src/containers/Surveys/Dashboard/Module/Statements/index.js

```diff
@@ -48,12 +51,20 @@ const StatementsContent = ({statementData, index, topic}) => {
         );
     }, [expanded]);
 
+    const handleExportPDF = useCallback(async () => {
+        if(!expanded) {
+            toggleExpand();
+        }
+        await sleep(80); //Allow all remaining renders to complete
+        window.print();
+    }, [toggleExpand, expanded]);
+
```

Comment:
Expand logic might have to be moved to the parent, since this only expands statements from the first topic.

Reply:
ohh okay